manifests/concourse-manifest/spec/support/manifest_helpers.rb

```diff
@@ -29,31 +40,43 @@ def concourse_secrets_value(key)
 
 private
 
+  def root
+    Pathname(File.expand_path("../../../..", __dir__))
+  end
+
   def fake_env_vars
-    ENV["AWS_ACCOUNT"] = "dev"
-    ENV["CONCOURSE_INSTANCE_TYPE"] = "t2.small"
-    ENV["CONCOURSE_INSTANCE_PROFILE"] = "concourse-build"
-    ENV["CONCOURSE_AUTH_DURATION"] = "5m"
-    ENV["SYSTEM_DNS_ZONE_NAME"] = ManifestHelpers::SYSTEM_DNS_ZONE_NAME
+    env = {}
+    env["AWS_ACCOUNT"] = "dev"
+    env["CONCOURSE_INSTANCE_TYPE"] = "t2.small"
+    env["CONCOURSE_INSTANCE_PROFILE"] = "concourse-build"
+    env["CONCOURSE_AUTH_DURATION"] = "5m"
+    env["SYSTEM_DNS_ZONE_NAME"] = ManifestHelpers::SYSTEM_DNS_ZONE_NAME
+    env["ENABLE_GITHUB"] = "false"
+    env
   end
 
-  def load_default_manifest
-    fake_env_vars
+  def render_manifest(override_env: {})
+    workdir = Pathname(Dir.mktmpdir('paas-bootstrap-test'))
+
+    env = fake_env_vars.merge(override_env)
+    env['PAAS_BOOTSTRAP_DIR'] = root.to_s
+    env['WORKDIR'] = workdir.to_s
+
+    copy_terraform_fixtures("#{workdir}/terraform-outputs", %w(vpc bosh concourse))
+    FileUtils.mkdir("#{workdir}/concourse-secrets")
+    FileUtils.cp(concourse_secrets_file, "#{workdir}/concourse-secrets/concourse-secrets.yml")
+
     output, error, status = Open3.capture3(
-      [
-        File.expand_path("../../../../shared/build_manifest.sh", __FILE__),
-        File.expand_path("../../../concourse-base.yml", __FILE__),
-        concourse_secrets_file,
-        File.expand_path("../../../../shared/spec/fixtures/concourse-terraform-outputs.yml", __FILE__),
-        File.expand_path("../../../../shared/spec/fixtures/bosh-terraform-outputs.yml", __FILE__),
-        File.expand_path("../../../../shared/spec/fixtures/vpc-terraform-outputs.yml", __FILE__),
-      ].join(' ')
+      env,
+      root.join("manifests/concourse-manifest/scripts/generate-manifest.sh").to_s,
     )
     expect(status).to be_success, "build_manifest.sh exited #{status.exitstatus}, stderr:\n#{error}"
```

Comment:
this should say `generate-manifest.sh exited`

Reply:
Fixed